cellacdc/workers.py

```diff
@@ -1511,9 +1512,9 @@ def run(self):
         trackerInputImage = None
         self.track_params['signals'] = self.signals
         if 'image' in self.track_params:
-            trackerInputImage = self.track_params.pop('image')
```

Comment:
Can you please reinsert `trackerInputImage = self.track_params.pop('image')` at line 1514? Thanks!